pinot-plugins/pinot-file-system/pinot-s3/src/main/java/org/apache/pinot/plugin/filesystem/S3PinotFS.java

```diff
@@ -161,6 +161,8 @@ public void init(PinotConfiguration config) {
         assert (_storageClass != StorageClass.UNKNOWN_TO_SDK_VERSION);
       }
 
+      s3ClientBuilder.crossRegionAccessEnabled(s3Config.isCrossRegionAccessEnabled());
```

Comment:
nit: why not include this in the builder method chain above?

Reply:
Sure, done!